Gemfile

```diff
@@ -0,0 +1,8 @@
+source "https://rubygems.org"
+
+group :development do
+  gem "test-kitchen", :git => 'https://github.com/test-kitchen/test-kitchen.git', :branch => 'windows-guest-support'
+  gem 'kitchen-vagrant', git: 'https://github.com/afiune/kitchen-vagrant.git', :branch => 'Transport'
+  gem "berkshelf"
+  gem "vagrant-wrapper", ">= 2.0"
```

Comment:
Since you've added chefspec stuff, you probably should probably add rspec and chefspec in here